### DISCUSSION TITLE
NavMesh/Pathfinding utilities

### DISCUSSION THREAD

**Discussion by InnovativeName-GameDev** [bevyengine/bevy#5350] [Ideas]
I just want to ask, if there is a plan to add 3d NavMesh/Pathfinding utilities in the near future?

**Comment by mockersf** [comment]
Included in Bevy? Not in the near future.
As a third party plugins? As soon as someone wants to do it 😄 

I did a small PoC more than one year ago: https://github.com/vleue/bevy_pathmesh (small video example there: https://twitter.com/FrancoisMockers/status/1373742786877534216). it should be quite easy to update and improve. I currently don't have a need for it but would love to build it more if someone wants to drive it with actual usage.

**Reply by InnovativeName-GameDev** [reply]
Well, currently im learning rust with bevy, coming from a Unity background, and my plan is first to learn a bit more with rust and bevy  and then to prototype a bit more, also porting my game (currently an early prototype) from Unity dots to bevy, but in like 3-6 months, i might have the need for it. Its good to know there is at least the chance for a pluggin. If the time comes i'll property wirte about it in this thread. Have a nice day.
